sync-main-transifex

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+git checkout master
+git checkout -b sync-main-transifex
+
+./autogen.sh --prefix=/usr
+
+rm po/*.po
```

Comment:
Why is this needed?
A pull writes new translations in existing po files.

Reply:
It removes unattended locales:
```shell
$ ./sync-transifex-translations
...
$ git diff po/LINGUAS
diff --git a/po/LINGUAS b/po/LINGUAS
index 86a2261..0f775ca 100644
--- a/po/LINGUAS
+++ b/po/LINGUAS
@@ -1,5 +1,4 @@
 # please keep this list sorted alphabetically
-#
 af
 am
 ar
@@ -30,15 +29,12 @@ es
 es_AR
 es_CL
 es_CO
-es_ES
 es_MX
-es_PR
 et
 eu
 fa
 fi
 fr
-fr_CA
 frp
 fur
 fy
@@ -59,7 +55,6 @@ ig
 is
 it
 ja
-jv
 ka
 kk
 kn
```

Reply:
Good, that makes sense.